client/knowledge_base_search/src/containers/SearchContainer/styles.css

```diff
@@ -1,11 +1,113 @@
 .header--input {
   color: #ffffff;
   text-align: center;
+  margin-bottom: 2rem;
+}
+
+.clickable-tabs {
+  display: flex;
+  border-spacing: 1rem;
+  width: 70rem;
+  position: relative;
+  bottom: 3rem;
+}
+
+.clickable-tab {
+  color: #ffffff;
+  border: 2px solid #f0f0f0;
+  border-radius: 0.5rem;
+  background-color: inherit;
+  padding: 1rem;
+  font-size: 0.9em;
+  font-weight: 200;
+  margin-top: -1rem;
+  margin-left: 1rem;
+  position: relative;
+  left: -4.9rem;
+  top: 3rem;
+  max-width: 11.6rem;
+  text-align: left;
+  vertical-align: middle;
+  min-height: 9rem;
+  line-height: 1.2rem;
+}
+
+.clickable-tab:hover {
+  transition: 0.4s;
+  background-color: #41d6c3;
+  border-color: #ffffff;
+  color: #003c32;
+}
+
+.active_button {
+  border-color: #ffffff;
+  color: #ffffff;
+  background-color: #008571;
+  outline: 0;
+}
+
+.tab-panels {
+  background: transparent;
+  border: none;
+}
+
+.tab-panels--tab.base--a {
+  color: #a7fae6;
+  font-weight: 200;
+  margin-right: 2rem;
+  width: 7.3rem;
+  border-bottom-width: 0.2rem;
+  padding: 0.5rem 0rem;
+}
+
+.tab-panels--tab.base--a:hover {
+  border-bottom-color: transparent;
+}
+
+.tab-panels--tab.active {
+  color: #fff;
+  border-bottom-color: #fff;
+}
+
+.tab-panels--tab:focus {
+  color: #fff !important;
```

Comment:
you should just be able to add another class to the element that you want to override these properties instead of using `!important` -> css specificity https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity